tests/storage/snapshot.go

```diff
@@ -1001,6 +1001,54 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 				}
 			})
 
+			It("[test_id:4611] should successfully create a snapshot", func() {
```

Comment:
Maybe this deserves a separate commit, a bit confusing as to why you'd want to put back a test that was under the GC branch

Reply:
@ShellyKa13 is there anything special about this test? from what I can tell it's probably something that is exercised already

Reply:
I also thought if this test should be removed, but I saw that it gets into volumes verification with volumesnapshots, which I dont think there is another test that checks that deep.. I figured that it probably existed before the DV GC and that it was converted to also be tested with GC..

Reply:
Right, I just reverted the GC split.